Lib/test/test_audit.py

```
@@ -18,7 +18,7 @@
 class AuditTest(unittest.TestCase):
     def do_test(self, *args):
         with subprocess.Popen(
-            [sys.executable, "-X utf8", AUDIT_TESTS_PY, *args],
+            [sys.executable, "-Xutf8", AUDIT_TESTS_PY, *args],
```

Bonus! This was a bug because the Xoption was provided as ` utf8` (with space) and not `utf8` so it was silently ignored :(

@vstinner @zooba

---

Oh, or you can write "-X", "utf8". I suppose that both work.